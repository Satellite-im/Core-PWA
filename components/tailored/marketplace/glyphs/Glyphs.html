<div class="marketplace-glyphs">
  <TailoredMarketplaceGlyphsTrending
    :list="$mock.glyphs"
  ></TailoredMarketplaceGlyphsTrending>
  <TailoredMarketplaceGlyphsShop></TailoredMarketplaceGlyphsShop>
</div>
