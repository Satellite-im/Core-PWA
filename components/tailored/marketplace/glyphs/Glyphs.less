.marketplace-glyphs {
  .subtitle {
    color: @bright-text;
  }
  .marketplace-glyphs-detail {
    .go-back {
      display: flex;
      align-items: center;
      cursor: pointer;
      color: @bright-text;
    }
  }
  .badges {
    display: flex;
    align-items: center;
    .badge-new,
    .badge-sale {
      color: @white;
      background-color: @green;
      padding: 8px 10px;
      border-radius: 4px;
      margin-right: @xlight-spacing;
    }
    .badge-new {
      background-color: @green;
    }
    .badge-sale {
      background-color: @red;
    }
  }
}
