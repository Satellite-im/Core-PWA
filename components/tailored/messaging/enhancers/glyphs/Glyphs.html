<div id="glyphs">
    <TailoredMessagingEnhancersGlyphsPackList></TailoredMessagingEnhancersGlyphsPackList>
</div>