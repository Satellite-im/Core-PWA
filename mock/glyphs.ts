import { Config } from '~/config'

export const Glyphs = {
  astrobunny: {
    name: 'Astrobunny',
    description: 'Short description can go here. Lorem ipsum.',
    artist: 'John Treanor',
    id: '0123',
    stickerURLs: [
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/AHH.gif`,
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/AHHcloseup.gif`,
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/Coy_02.gif`,
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/Cry.gif`,
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/Cry2.gif`,
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/Laugh.png`,
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/Luv.gif`,
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/Luv_02.gif`,
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/Sad3.gif`,
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/ThumbsDownNew.48.gif`,
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/ThumbsUP.gif`,
      `${Config.ipfs.gateway}QmaBFFeJksvrfEhwbKYkzbgm7bKFWzHhRECqGxgB52LeHg/ThumbsUPNew.gif`,
    ],
  },
  grimreaper: {
    name: 'Grim Reaper',
    description: 'Short description can go here. Lorem ipsum.',
    artist: 'LAM',
    id: '012312',
    stickerURLs: [
      `${Config.ipfs.gateway}QmUk5XGfk4Bf6DPq8WZUz6bWqw8HKBxCjPgUVFy7hois1T/1.png`,
      `${Config.ipfs.gateway}QmUk5XGfk4Bf6DPq8WZUz6bWqw8HKBxCjPgUVFy7hois1T/2.png`,
      `${Config.ipfs.gateway}QmUk5XGfk4Bf6DPq8WZUz6bWqw8HKBxCjPgUVFy7hois1T/3.png`,
      `${Config.ipfs.gateway}QmUk5XGfk4Bf6DPq8WZUz6bWqw8HKBxCjPgUVFy7hois1T/4.png`,
      `${Config.ipfs.gateway}QmUk5XGfk4Bf6DPq8WZUz6bWqw8HKBxCjPgUVFy7hois1T/5.png`,
      `${Config.ipfs.gateway}QmUk5XGfk4Bf6DPq8WZUz6bWqw8HKBxCjPgUVFy7hois1T/6.png`,
    ],
  },
  marshmellow: {
    name: 'Marshmello',
    description: 'Short description can go here. Lorem ipsum.',
    artist: 'John Treanor',
    id: '012312c',
    stickerURLs: [
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Ahhhhh.gif`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_01.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_02.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_03.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_04.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_05.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_06.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_07.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_08.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_09.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_10.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_11.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_12.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Emoji_13.png`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/HellYea.gif`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/Luv.gif`, 
      `${Config.ipfs.gateway}QmYBXXrrHNC61f72xsGgTdnq8uTcGLW6a7LG5NFmRLJgwL/spark.gif`, 
    ],
  },
  food: {
    name: 'Food',
    description: 'Short description can go here. Lorem ipsum.',
    artist: 'Lya',
    id: 'jdsoer',
    stickerURLs: [
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/Beer.gif`, 
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/Toast.gif`, 
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/chocolate_angry.gif`, 
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/chocolate_cute.gif`, 
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/coconut_cute.gif`, 
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/coconut_rude.gif`, 
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/coffee_cute.gif`, 
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/coffee_rude.gif`, 
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/croissant2.gif`, 
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/croissant1.gif`, 
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/fries.gif`, 
      `${Config.ipfs.gateway}QmRowCbn4PrZFhHArXJeRSBqEoqtFm6koEjEkKWEzWn8Tz/onigiri.gif`, 
    ],
  },
  blackcat: {
    name: 'Black Cat',
    description: 'Short description can go here. Lorem ipsum.',
    artist: 'Dan Tavis',
    id: 'k321r',
    stickerURLs: [
      `${Config.ipfs.gateway}QmZ1SXaQNsEn4bsCdh2hLmYfUe5uDeYP2RqiwhZMvALqNE/Black-Cat-Attention-2.png`, 
      `${Config.ipfs.gateway}QmZ1SXaQNsEn4bsCdh2hLmYfUe5uDeYP2RqiwhZMvALqNE/Cat-+-Flowers.png`, 
      `${Config.ipfs.gateway}QmZ1SXaQNsEn4bsCdh2hLmYfUe5uDeYP2RqiwhZMvALqNE/Cat-Vs-Yarn-%28cover%29.png`, 
      `${Config.ipfs.gateway}QmZ1SXaQNsEn4bsCdh2hLmYfUe5uDeYP2RqiwhZMvALqNE/Fluff.png`, 
      `${Config.ipfs.gateway}QmZ1SXaQNsEn4bsCdh2hLmYfUe5uDeYP2RqiwhZMvALqNE/Sleep-Day%20%284%29.png`, 
      `${Config.ipfs.gateway}QmZ1SXaQNsEn4bsCdh2hLmYfUe5uDeYP2RqiwhZMvALqNE/Thumbs-Up-Cat.png`, 
    ],
  },
  genshin: {
    name: 'Genshin Impact 1',
    description: 'Short description can go here. Lorem ipsum.',
    artist: 'John Treanor',
    id: '001',
    stickerURLs: [
      `${Config.ipfs.gateway}QmS8UaEimprq4P37ehjDDj7wJ8if3SpjUHfXbXwBfBw5fi/JohnTreanor_GenshinImpactFood_01.png`,
      `${Config.ipfs.gateway}QmS8UaEimprq4P37ehjDDj7wJ8if3SpjUHfXbXwBfBw5fi/JohnTreanor_GenshinImpactFood_02.png`,
      `${Config.ipfs.gateway}QmS8UaEimprq4P37ehjDDj7wJ8if3SpjUHfXbXwBfBw5fi/JohnTreanor_GenshinImpactFood_03.png`,
      `${Config.ipfs.gateway}QmS8UaEimprq4P37ehjDDj7wJ8if3SpjUHfXbXwBfBw5fi/JohnTreanor_GenshinImpactFood_04.png`,
      `${Config.ipfs.gateway}QmS8UaEimprq4P37ehjDDj7wJ8if3SpjUHfXbXwBfBw5fi/JohnTreanor_GenshinImpactFood_05.png`,
      `${Config.ipfs.gateway}QmS8UaEimprq4P37ehjDDj7wJ8if3SpjUHfXbXwBfBw5fi/JohnTreanor_GenshinImpactFood_06.png`,
      `${Config.ipfs.gateway}QmS8UaEimprq4P37ehjDDj7wJ8if3SpjUHfXbXwBfBw5fi/JohnTreanor_GenshinImpactFood_07.png`,
      `${Config.ipfs.gateway}QmS8UaEimprq4P37ehjDDj7wJ8if3SpjUHfXbXwBfBw5fi/JohnTreanor_GenshinImpactFood_08.png`,
      `${Config.ipfs.gateway}QmS8UaEimprq4P37ehjDDj7wJ8if3SpjUHfXbXwBfBw5fi/JohnTreanor_GenshinImpactFood_09.png`,
      `${Config.ipfs.gateway}QmS8UaEimprq4P37ehjDDj7wJ8if3SpjUHfXbXwBfBw5fi/JohnTreanor_GenshinImpactFood_22.png`,
      `${Config.ipfs.gateway}QmS8UaEimprq4P37ehjDDj7wJ8if3SpjUHfXbXwBfBw5fi/JohnTreanor_GenshinImpactFood_23.png`,
    ],
  },
  genshinTwo: {
    name: 'Genshin Impact 2',
    description: 'Short description can go here. Lorem ipsum.',
    artist: 'John Treanor',
    id: '002',
    stickerURLs: [
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_10.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_11.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_12.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_13.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_14.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_15.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_16.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_17.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_18.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_19.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_20.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_22.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_21.png`, 
      `${Config.ipfs.gateway}QmRBxCyQa2yJFpF4yiV2wimLX2i32oty8yKFJFsFhyJR2n/JohnTreanor_GenshinImpactFood_23.png`, 
    ],
  },
  lamsEmoji: {
    name: 'LAM Collection',
    description: 'Short description can go here. Lorem ipsum.',
    artist: 'LAM',
    id: '0103',
    stickerURLs: [
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-1.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-3.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-4.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-5.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-6.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-7.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-8.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-9.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-10.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-11.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-13.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-14.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-15.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-16.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-17.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-18.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-19.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-20.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-21.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-22.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-23.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-122.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-612.png`, 
      `${Config.ipfs.gateway}QmWdzaL2gUbdAZKLhwBFbiToNmYq9Bt6ALofpSpw3Aiuso/Untitled-6123.png`, 
    ],
  },
  starWars: {
    name: 'Star Wars',
    description: 'Short description can go here. Lorem ipsum.',
    artist: 'John Treanor',
    id: '0053',
    stickerURLs: [
      `${Config.ipfs.gateway}QmcgsWnhMUpactZZQk9CSQ9iXc7cri1aCcDbs6DEfVgcom/JohnTreanor_StarWarsPack_01.png`,
      `${Config.ipfs.gateway}QmcgsWnhMUpactZZQk9CSQ9iXc7cri1aCcDbs6DEfVgcom/JohnTreanor_StarWarsPack_02.png`,
      `${Config.ipfs.gateway}QmcgsWnhMUpactZZQk9CSQ9iXc7cri1aCcDbs6DEfVgcom/JohnTreanor_StarWarsPack_03.png`,
      `${Config.ipfs.gateway}QmcgsWnhMUpactZZQk9CSQ9iXc7cri1aCcDbs6DEfVgcom/JohnTreanor_StarWarsPack_04.png`,
      `${Config.ipfs.gateway}QmcgsWnhMUpactZZQk9CSQ9iXc7cri1aCcDbs6DEfVgcom/JohnTreanor_StarWarsPack_05.png`,
      `${Config.ipfs.gateway}QmcgsWnhMUpactZZQk9CSQ9iXc7cri1aCcDbs6DEfVgcom/JohnTreanor_StarWarsPack_06.png`,
      `${Config.ipfs.gateway}QmcgsWnhMUpactZZQk9CSQ9iXc7cri1aCcDbs6DEfVgcom/JohnTreanor_StarWarsPack_07.png`,
      `${Config.ipfs.gateway}QmcgsWnhMUpactZZQk9CSQ9iXc7cri1aCcDbs6DEfVgcom/JohnTreanor_StarWarsPack_08.png`,
      `${Config.ipfs.gateway}QmcgsWnhMUpactZZQk9CSQ9iXc7cri1aCcDbs6DEfVgcom/JohnTreanor_StarWarsPack_09.png`,
    ],
  },
  birds: {
    name: 'Birds',
    description: 'Short description can go here. Lorem ipsum.',
    artist: 'Dina Brodsky',
    id: '0903',
    stickerURLs: [
      `${Config.ipfs.gateway}QmXrmiCxkyfpsAY18qzVBPv9TpCbTUToowQEsYy1Pm5C9b/hawk.png`,
      `${Config.ipfs.gateway}QmXrmiCxkyfpsAY18qzVBPv9TpCbTUToowQEsYy1Pm5C9b/ducklings.png`,
      `${Config.ipfs.gateway}QmXrmiCxkyfpsAY18qzVBPv9TpCbTUToowQEsYy1Pm5C9b/owl.png`,
      `${Config.ipfs.gateway}QmXrmiCxkyfpsAY18qzVBPv9TpCbTUToowQEsYy1Pm5C9b/penguins.png`,
      `${Config.ipfs.gateway}QmXrmiCxkyfpsAY18qzVBPv9TpCbTUToowQEsYy1Pm5C9b/robin.png`,
      `${Config.ipfs.gateway}QmXrmiCxkyfpsAY18qzVBPv9TpCbTUToowQEsYy1Pm5C9b/stork.png`,
      `${Config.ipfs.gateway}QmXrmiCxkyfpsAY18qzVBPv9TpCbTUToowQEsYy1Pm5C9b/yellow_bird.png`,
    ],
  },
}
