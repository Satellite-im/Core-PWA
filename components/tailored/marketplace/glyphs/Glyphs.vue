<template src="./Glyphs.html"></template>

<script lang="ts">
import Vue from 'vue'
import { mapState } from 'vuex'
import { ChevronLeftIcon } from 'satellite-lucide-icons'
import { marketGlyphs } from '~/mock/marketplace'
import { GlyphMarketViewStatus } from '~/store/ui/types'

export default Vue.extend({
  components: {
    ChevronLeftIcon,
  },
  props: {},
  data() {
    return {
      marketGlyphs,
    }
  },
  computed: {
    GlyphMarketViewStatus: () => GlyphMarketViewStatus,
    ...mapState(['ui']),
  },
  watch: {
    '$store.state.ui.glyphMarketplaceView'() {
      this.$parent.$el.scrollTop = 0
    },
  },
  methods: {
    goBack() {
      this.$store.commit('ui/setGlyphMarketplaceView', {
        view: GlyphMarketViewStatus.HOME,
        shopId: null,
      })
    },
  },
})
</script>

<style lang="less" src="./Glyphs.less"></style>
