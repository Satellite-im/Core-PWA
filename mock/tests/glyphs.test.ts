import { expect } from '@jest/globals'
import { Glyphs } from '../glyphs'

describe('init', () => {
  it('gets the constant', () => {
    expect(Glyphs).toMatchSnapshot()
  })
})
