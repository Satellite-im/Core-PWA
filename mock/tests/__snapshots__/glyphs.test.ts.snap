// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init gets the constant 1`] = `
Object {
  "astrobunny": Object {
    "artist": "John Treanor",
    "description": "Short description can go here. Lorem ipsum.",
    "id": "0123",
    "name": "Astrobunny",
    "stickerURLs": Array [
      "glyphs/astrobunny/$1/AHH.gif",
      "glyphs/astrobunny/$1/AHHcloseup.gif",
      "glyphs/astrobunny/$1/Coy_02.gif",
      "glyphs/astrobunny/$1/Cry.gif",
      "glyphs/astrobunny/$1/Cry2.gif",
      "glyphs/astrobunny/$1/Laugh.webp",
      "glyphs/astrobunny/$1/Luv.gif",
      "glyphs/astrobunny/$1/Luv_02.gif",
      "glyphs/astrobunny/$1/Sad3.gif",
      "glyphs/astrobunny/$1/ThumbsDownNew.gif",
      "glyphs/astrobunny/$1/ThumbsUP.gif",
      "glyphs/astrobunny/$1/ThumbsUPNew.gif",
    ],
  },
  "birds": Object {
    "artist": "Dina Brodsky",
    "description": "Short description can go here. Lorem ipsum.",
    "id": "0903",
    "name": "Birds",
    "stickerURLs": Array [
      "glyphs/birds/$1/hawk.webp",
      "glyphs/birds/$1/ducklings.webp",
      "glyphs/birds/$1/owl.webp",
      "glyphs/birds/$1/penguins.webp",
      "glyphs/birds/$1/robin.webp",
      "glyphs/birds/$1/stork.webp",
      "glyphs/birds/$1/yellow_bird.webp",
    ],
  },
  "blackcat": Object {
    "artist": "Dan Tavis",
    "description": "Short description can go here. Lorem ipsum.",
    "id": "k321r",
    "name": "Black Cat",
    "stickerURLs": Array [
      "glyphs/black_cat/$1/Black-Cat-Attention-2.webp",
      "glyphs/black_cat/$1/Cat-+-Flowers.webp",
      "glyphs/black_cat/$1/Cat-Vs-Yarn-%28cover%29.webp",
      "glyphs/black_cat/$1/Fluff.webp",
      "glyphs/black_cat/$1/Sleep-Day%20%284%29.webp",
      "glyphs/black_cat/$1/Thumbs-Up-Cat.webp",
    ],
  },
  "food": Object {
    "artist": "Lya",
    "description": "Short description can go here. Lorem ipsum.",
    "id": "jdsoer",
    "name": "Food",
    "stickerURLs": Array [
      "glyphs/lya/$1/Beer.gif",
      "glyphs/lya/$1/Toast.gif",
      "glyphs/lya/$1/chocolate_angry.gif",
      "glyphs/lya/$1/chocolate_cute.gif",
      "glyphs/lya/$1/coconut_cute.gif",
      "glyphs/lya/$1/coconut_rude.gif",
      "glyphs/lya/$1/coffee_cute.gif",
      "glyphs/lya/$1/coffee_rude.gif",
      "glyphs/lya/$1/croissant2.gif",
      "glyphs/lya/$1/croissant1.gif",
      "glyphs/lya/$1/fries.gif",
      "glyphs/lya/$1/onigiri.gif",
    ],
  },
  "genshin": Object {
    "artist": "John Treanor",
    "description": "Short description can go here. Lorem ipsum.",
    "id": "001",
    "name": "Genshin Impact 1",
    "stickerURLs": Array [
      "glyphs/genshin_1/$1/JohnTreanor_GenshinImpactFood_01.webp",
      "glyphs/genshin_1/$1/JohnTreanor_GenshinImpactFood_02.webp",
      "glyphs/genshin_1/$1/JohnTreanor_GenshinImpactFood_03.webp",
      "glyphs/genshin_1/$1/JohnTreanor_GenshinImpactFood_04.webp",
      "glyphs/genshin_1/$1/JohnTreanor_GenshinImpactFood_05.webp",
      "glyphs/genshin_1/$1/JohnTreanor_GenshinImpactFood_06.webp",
      "glyphs/genshin_1/$1/JohnTreanor_GenshinImpactFood_07.webp",
      "glyphs/genshin_1/$1/JohnTreanor_GenshinImpactFood_08.webp",
      "glyphs/genshin_1/$1/JohnTreanor_GenshinImpactFood_09.webp",
    ],
  },
  "genshinTwo": Object {
    "artist": "John Treanor",
    "description": "Short description can go here. Lorem ipsum.",
    "id": "002",
    "name": "Genshin Impact 2",
    "stickerURLs": Array [
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_10.webp",
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_11.webp",
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_12.webp",
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_13.webp",
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_14.webp",
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_15.webp",
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_16.webp",
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_17.webp",
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_18.webp",
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_19.webp",
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_20.webp",
      "glyphs/genshin_2/$1/JohnTreanor_GenshinImpactFood_21.webp",
    ],
  },
  "grimreaper": Object {
    "artist": "LAM",
    "description": "Short description can go here. Lorem ipsum.",
    "id": "012312",
    "name": "Grim Reaper",
    "stickerURLs": Array [
      "glyphs/grim_reaper/$1/1.webp",
      "glyphs/grim_reaper/$1/2.webp",
      "glyphs/grim_reaper/$1/3.webp",
      "glyphs/grim_reaper/$1/4.webp",
      "glyphs/grim_reaper/$1/5.webp",
      "glyphs/grim_reaper/$1/6.webp",
    ],
  },
  "marshmellow": Object {
    "artist": "John Treanor",
    "description": "Short description can go here. Lorem ipsum.",
    "id": "012312c",
    "name": "Marshmello",
    "stickerURLs": Array [
      "glyphs/marshmello/$1/Ahhhhh.gif",
      "glyphs/marshmello/$1/Emoji_01.webp",
      "glyphs/marshmello/$1/Emoji_02.webp",
      "glyphs/marshmello/$1/Emoji_03.webp",
      "glyphs/marshmello/$1/Emoji_04.webp",
      "glyphs/marshmello/$1/Emoji_05.webp",
      "glyphs/marshmello/$1/Emoji_06.webp",
      "glyphs/marshmello/$1/Emoji_07.webp",
      "glyphs/marshmello/$1/Emoji_08.webp",
      "glyphs/marshmello/$1/Emoji_09.webp",
      "glyphs/marshmello/$1/Emoji_10.webp",
      "glyphs/marshmello/$1/Emoji_11.webp",
      "glyphs/marshmello/$1/Emoji_12.webp",
      "glyphs/marshmello/$1/Emoji_13.webp",
      "glyphs/marshmello/$1/HellYea.gif",
      "glyphs/marshmello/$1/Luv.gif",
      "glyphs/marshmello/$1/spark.gif",
    ],
  },
  "starWars": Object {
    "artist": "John Treanor",
    "description": "Short description can go here. Lorem ipsum.",
    "id": "0053",
    "name": "Star Wars",
    "stickerURLs": Array [
      "glyphs/star_wars/$1/JohnTreanor_StarWarsPack_01.webp",
      "glyphs/star_wars/$1/JohnTreanor_StarWarsPack_02.webp",
      "glyphs/star_wars/$1/JohnTreanor_StarWarsPack_03.webp",
      "glyphs/star_wars/$1/JohnTreanor_StarWarsPack_04.webp",
      "glyphs/star_wars/$1/JohnTreanor_StarWarsPack_05.webp",
      "glyphs/star_wars/$1/JohnTreanor_StarWarsPack_06.webp",
      "glyphs/star_wars/$1/JohnTreanor_StarWarsPack_07.webp",
      "glyphs/star_wars/$1/JohnTreanor_StarWarsPack_08.webp",
      "glyphs/star_wars/$1/JohnTreanor_StarWarsPack_09.webp",
    ],
  },
}
`;
